tests/types/rankn/impredicative01-fail.out

```diff
@@ -1,6 +1,3 @@
-impredicative01-fail.ml[6:38 ..6:52]: error
-  │ 
-6 │ let foo : id (forall 'a. 'a -> 'a) = Id (fun x -> x)
-  │                                      ^^^^^^^^^^^^^^^
-  Couldn't match actual type id ('ah -> 'ah)
-    with the type expected by the context, id (forall 'a. 'a -> 'a)
+id : type -> type
```

Comment:
Can we re{name,do,move} this test, if it no longer fails?

Reply:
sure